common/scripted_effects/00_animal_effects.txt

```diff
@@ -599,14 +899,32 @@ transfer_dog_story_cycle_to_effect = {
 		add_character_modifier = {
 			modifier = dog_story_modifier
 		}
+		if = {
+			limit = {
+				scope:original_owner = {
+					has_character_modifier = raven_companion
+				}
+			}
+			add_character_modifier = raven_companion
+		}
+		else_if = {
+			limit = {
+				scope:original_owner = {
+					has_character_modifier = bear_companion
+				}
+			}
+			add_character_modifier = bear_companion
+		}
```

Comment:
That's not the full list of companions. Why?

Reply:
Because I forgot that I needed to change that, whoops